### PR TITLE
Mainnet Folder Updates for EigenDA Operator Instructions and config

### PR DESCRIPTION
**Pull request by wesfloyd**
**Due Date**: Monday 4/8

For this PR I've:

- Copied the Holesky folder content.
- Replaced occurrence of the words Holesky, Testnet, etc. with Mainnet and other reasonable fixes I was able to identify.

Todos:
- [x] Update remaining todos in the .env file: 2x contract addresses and NODE_CHURNER_URL

Todo: after this PR is merged.
- [ ] Submit a followup PR, modify the relative location file references to "./.env.example" file in mainnet/Readme.md to point to the static permalink for the file instead.

### CODE REVIEW

File: mainnet/README.md [hunk 1]
```diff
@@ -1,2 +1,215 @@
...
+```
+
+##### Step 3.4 Test the values of your environment variables $USER_HOME, $EIGENLAYER_HOME, $EIGENDA_HOME to ensure all the folders exist as expected. Review the resulting values Correct any folders whose paths are not set as expected.
+
+```
+source .env
+ls $USER_HOME $EIGENLAYER_HOME $EIGENDA_HOME
+```
+
+#### Step 4: Operator Networking Security Setup
```
Comment (shrimalmadhur) [inline mainnet/README.md:42]: I realized we need to put another port in here for operators to open.  in addition to dispersal and retrieval - we should ask them to open `NODE_API_PORT` to internet so that we can get some basic node data from them. I know we didn't ask on holesky but we should add it. So we can say something like

Node API port: 
In order to get some basic data please also open `NODE_API_PORT` in [.env](./.env.example#L16) to internet. The data which we will get from that API is mentioned in the [Node API Spec](https://docs.eigenlayer.xyz/eigenlayer/avs-guides/spec/api/)

Comment (shrimalmadhur) [reply mainnet/README.md:42]: maybe add this in holesky too with this PR?

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (shrimalmadhur) [inline yarn.lock:1]: why is this added? 

same for yarn integrity

File: mainnet/README.md (hunk 1 above)
Comment (eigensid) [reply mainnet/README.md:42]: Is it mandatory for operators to open this port to Internet ingress?

Comment (shrimalmadhur) [reply mainnet/README.md:42]: it's not mandatory for eigenda working but we want them to open it - so that we can get some more data in terms of what versions they are running and eventually more details on all operator set.

File: yarn.lock (hunk 2 above)
Comment (wesfloyd) [reply yarn.lock]: Good catch! That was a mistake, I tried to run yarn, yarn build and then remembered this isn't a docusaurus repo :)

Comment (wesfloyd) [reply yarn.lock]: Removed both in the latest commit

**Comment by wesfloyd** [comment]
Reminder to Wes: [include these changes](docs/eigenlayer/restaking-guides/0-restaking-user-guide/stage-2-testnet/testnet-native-restaking/withdraw-from-eigenlayer/full-withdrawal.md) when merging the PR.

File: mainnet/README.md (hunk 1 above)
Comment (wesfloyd) [reply mainnet/README.md:42]: @shrimalmadhur does opening that port create any security issues for Operators? Or perhaps "not" because its only a read-only API?

Does the following revised text work for you?
> Node API port:
In order to consolidate Operator metrics to measure the health of the network, please also open NODE_API_PORT in [.env](https://github.com/Layr-Labs/eigenda-operator-setup/pull/.env.example#L16) to the internet if possible. Please see [Node API Spec](https://docs.eigenlayer.xyz/eigenlayer/avs-guides/spec/api/) for more detail on the data made available via this port.

Comment (shrimalmadhur) [reply mainnet/README.md:42]: yea it's a readonly port. nothing I see as a concern. yea let's just make it optional. the updated text looks real good. thanks

Comment (shrimalmadhur) [reply mainnet/README.md:42]: once you add this I am good to go with this PR

Comment (wesfloyd) [reply mainnet/README.md:42]: @shrimalmadhur Node API port content has been added to this PR.

Please approve (and Merge on my behalf since I don't have permissions) when ready.

File: mainnet/run.sh [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    ghcr.io/layr-labs/eigenda/opr-nodeplugin:release-0.6.0 \
+    --ecdsa-key-password "$NODE_ECDSA_KEY_PASSWORD" \
+    --bls-key-password "$NODE_BLS_KEY_PASSWORD" \
+    --socket "$socket" \
+    --operation list-quorums \
+    --quorum-id-list 0
+}
+
+
+if [ "$1" = "opt-in" ]; then
```
Comment (shrimalmadhur) [inline mainnet/run.sh:64]: @wesfloyd could you add this change in here too https://github.com/Layr-Labs/eigenda-operator-setup/pull/97/files - let me know if you need help or if you want me to push to your PR

Comment (wesfloyd) [reply mainnet/run.sh:64]: Hi @shrimalmadhur would you mind pushing the commit to this PR directly? It would help me avoid any mistakes (assumptions) on the specific lines that should be added

Comment (shrimalmadhur) [reply mainnet/run.sh:64]: I will do it in separate PR.

**Review by shrimalmadhur — APPROVED** [review]
